apps/safe-claiming-app/src/hooks/useAmounts.ts

```diff
@@ -9,23 +11,33 @@ export const useAmounts = (
   const [claimableAmount, setClaimableAmount] = useState("0")
   const [amountInVesting, setAmountInVesting] = useState("0")
   const [currentIntervalId, setCurrentIntervalId] = useState<number>()
+  const { safe, sdk } = useSafeAppsSDK()
+  const web3Provider = getWeb3Provider(safe, sdk)
```

Comment:
Since this is not a hook I think we can move it into the useEffect and also add `safe` and `sdk` as dependencies.